docs/ensembl-help/using-ensembl/ensembl-apps/genome-browser/tracks.md

```diff
@@ -13,9 +13,39 @@ status: draft
 
 # Tracks in the Genome browser
 
-Tracks are sets of data that can be plotted against the genome, such as transcripts, genetic variation or conserved regions. Entities within tracks are shown on horizontal rows. There may be multiple rows for a track, ensuring that there is no overlap within a row. Some tracks, such as genes, are associated with one strand of the genome – behind stranded tracks you will see small arrows pointing in the direction of transcription.
+## What is a track ? 
 
-You can modify the appearance of tracks by clicking on the cog icon to the right of the track. This allows you to show or hide labels on the track and/or the entity, to change the height of the track, to highlight it, to move it and to lock it.
+Tracks are sets of data such as genes, variants or regulatory motifs displayed along a genome. A track displays a single set of data ie genes on the forward strand.
 
-You can change which tracks you see using the Track panel on the right of the page. There are three categories of tracks: Genomic, Variation and Expression. Details of your Focus entity are always shown at the top of the track panel. Show or hide tracks using the eye icon. Find out more about a track or an entity by clicking on the ellipsis to open the drawer with more information and links to the Entity viewer. Use the arrow at the top of the Action bar to close the Track panel drawer or hide the Track panel completely.
+Individual entities ie genes, variants or regulatory regions within a track are shown on horizontal rows. There may be multiple rows within a track, ensuring that there is no entity overlap within a row. 
+
+Forward and reverse strands
+
+Some tracks, such as genes, are associated with a specific strand of the genome. Strands are represented by small arrows pointing in the direction of transcription set behind the track information. 
```

Comment:
'direction of transcription' is too narrow a description maybe ? A more general term would be 'direction of the assembly set behind the track information'